resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/usecase/SnapshotTest.java

```diff
@@ -73,7 +74,7 @@ public void test3() {
 		test.testSaveWorkspace();
 	}
 
-	public void test4() {
+	public void test4() throws CoreException {
 		if (skipTest()) {
```

Comment:
Use assumptions to skip this test visibly. It should be possible with JUnit4. If not, check whether or not you need to annotate the class with `@RunWith(JUnit4.class)`

Reply:
see above